deadlock/deadlock.tex

```diff
@@ -34,34 +34,34 @@ \section{Resource Allocation Graphs}
 If a process is \emph{using} a resource, an arrow is drawn from the resource node to the process node.
 If a process is \emph{requesting} a resource, an arrow is drawn from the process node to the resource node.
 If there is a cycle in the Resource Allocation Graph and each resource in the cycle provides only one instance, then the processes will deadlock.
-For example, if process 1 holds resource A, process 2 holds resource B and process 1 is waiting for B and process 2 is waiting for A, then process 1 and 2 will be deadlocked \ref{ragfigure}.
-We'll make the distinction that the system is deadlocked by definition if all workers cannot perform an operation other than waiting.
+For example, if process 1 holds resource A, process 2 holds resource B and process 1 is waiting for B and process 2 is waiting for A, then processes 1 and 2 will be deadlocked \ref{ragfigure}.
+We'll make the distinction that the system is deadlocked by definition if all workers cannot operate other than waiting.
```

Comment:
This sentence doesn't make sense. `if all workers cannot operate other than waiting`